admin/jsonadm/src/Admin/JsonAdm/Order/Standard.php

```diff
@@ -203,6 +203,10 @@ protected function getChildItems( \Aimeos\Map $items, array $include ) : \Aimeos
 			$list = $list->merge( $baseItems )
 				->merge( $baseItems->getAddresses()->flat() )->merge( $baseItems->getCoupons()->flat() )
 				->merge( $baseItems->getProducts()->flat() )->merge( $baseItems->getServices()->flat() );
+
+			foreach($baseItems->getServices()->flat() as $row) {
+				$list->merge( $row->getAttributeItems()->flat() );
+			}
```

Comment:
You can shorten that to (and include product attributes too):
```
->merge( $baseItems->getProducts()->flat()->getAttributeItems() )
->merge( $baseItems->getServices()->flat()->getAttributeItems() );
```

Reply:
When i try to shorten it

It throw following error:
ErrorException: Illegal offset type in isset or empty in file admin/jsonadm/templates/partials/order/included-standard.php on line 39

however it works when using foreach loop.

I'm not sure what's going on there

Reply:
The principle is explained in the "jQuery style" section of https://php-map.org/#why-php-map
But I'm also not sure what's the problem. In the `foreach()` loop, you can at least remove the `flat()` call because the result of `getAttributeItems()` is already a flat array.